src/components/SelectField/SelectField.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Select, P } from './style';
+
+const SelectField = (props) => {
+  const {
+    options, onChange, defaultText, onBlur, error,
+  } = props;
+  return (
+    <div>
+      <Select name="Sport" onChange={onChange} onBlur={onBlur}>
```

Comment:
Why use hard-code value (name="Sport") ?
Accept a prop (value or name) and use for the select